src/modules/market/components/market-orders-positions-table/open-orders-table--orders.jsx

```diff
@@ -89,30 +90,32 @@ export default class OpenOrdersOrder extends Component {
           style={{ textTransform: "capitalize" }}
         >
           {orderType}
+          {order.pendingOrder && <span>Pending</span>}
         </li>
         <li>{orderShares}</li>
         <li>{orderPrice}</li>
         {outcome && <li>{getValue(outcome, "lastPrice.formatted")}</li>}
         {<li>{getValue(order, "tokensEscrowed.formatted")}</li>}
         {<li>{getValue(order, "sharesEscrowed.formatted")}</li>}
-        {!isMobile && (
-          <li>
-            {pending ? (
-              <button className={Styles.Order__cancel} disabled>
-                PENDING
-              </button>
-            ) : (
-              <button
-                className={Styles.Order__cancel}
-                onClick={e => {
-                  order.cancelOrder(order);
-                }}
-              >
-                Cancel
-              </button>
-            )}
-          </li>
-        )}
+        {!isMobile &&
+          order.cancelOrder && (
+            <li>
+              {pending ? (
+                <button className={Styles.Order__cancel} disabled>
+                  PENDING
+                </button>
+              ) : (
+                <button
+                  className={Styles.Order__cancel}
+                  onClick={e => {
+                    order.cancelOrder(order);
+                  }}
+                >
+                  Cancel
+                </button>
+              )}
```

Comment:
this should be a common component imo
cancel order buttons are all over the place in new redesign